mPower2/mPower2/StudyBurstScheduleManager.swift

```diff
@@ -852,6 +852,17 @@ class StudyBurstScheduleManager : TaskGroupScheduleManager {
         return (requests, pendingRequestIds)
     }
     
+    func willPresentNotification(_ notification: UNNotification) {
+        let nextDate = notification.date.addingNumberOfDays(90)
```

Comment:
Should we check first to see if their time in the study is up (2 years IIRC?) before setting the next study burst reminder? Or did we cover this in the group discussion, I don't remember

Reply:
Right, I don't remember either. I'll go ahead and add that.